components/ui/LatestLeadsList/LatestLeadsList.tsx

```diff
@@ -0,0 +1,48 @@
+import { LEADS_DATA } from '@/utils/constants/leadslist-constant';
+import ViewAllLeadsButton from '../ViewAllLeadsButton/ViewAllLeadsButton';
+import LatestLeadRow from '../LatestLeadRow/LatestLeadRow';
+import { getLatestLeads } from '@/utils/helpers/common-helpers';
+
+function LatestLeadsList() {
+  
+
+  return (
+    <>
```

Comment:
I hope, here it is not necessary to keep this fragment

Reply:
Done